app/cloudinit/cloudinit.go

```diff
@@ -559,39 +637,40 @@ var amazonLinuxTemplate = template.Must(template.New(oshelp.OSLinux).Funcs(funcs
 // Linux creates a userdata file for the Linux operating system.
 func Linux(params *Params) (payload string) {
 	sb := &strings.Builder{}
-
 	caCertPath := filepath.Join(certsDir, "ca-cert.pem")
 	certPath := filepath.Join(certsDir, "server-cert.pem")
 	keyPath := filepath.Join(certsDir, "server-key.pem")
+	p := struct {
+		Params
+		CaCertPath string
```

Comment:
CACertPath

Reply:
I just moved this outside to reuse it. CaCertPath is already being used in the template, would have to change it there as well. As of now, we are avoiding changes in existing CI code.